_faqs/03-12-are-there-any-changes-required-for-my-existing-elasticsearch-clients-to-continue-to-work.markdown

```diff
@@ -0,0 +1,5 @@
+---
+question: Are there any changes required for my existing Elasticsearch OSS clients to continue to work?
+category: Upgrading
+---
+While the OpenSearch API is backwards compatible, some clients or tools may include code, such as version checks, that may cause the client or tool to not work with OpenSearch. 
```

Comment:
If we decide to keep the Wire compatibility FAQ we should update it to include this as a note. It's good to call out.

Reply:
Added.